lib/requests/request_helpers.rb

```diff
@@ -129,11 +132,13 @@ def to_json
 
   class NodeSelector
```

Comment:
Please align this class with JVM counterpart: https://github.com/ravendb/ravendb-jvm-client/blob/v4.0/src/main/java/net/ravendb/client/http/NodeSelector.java

Reply:
I will do this in a separate PR (RDBC-169).